dev-support/hbase_docker/Dockerfile

```diff
@@ -14,37 +14,73 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:14.04
+FROM ubuntu:18.04 AS BASE_IMAGE
```

Comment:
rebuilt this one following the multi-stage pattern from the Dockerfile use in CI jobs.